lib/i18n/tasks/scanners/ruby_key_literals.rb

```diff
@@ -20,7 +20,7 @@ def strip_literal(literal)
       literal
     end
 
-    VALID_KEY_CHARS = %r{(?:[[:word:]]|[-.?!:;À-ž/])}.freeze
+    VALID_KEY_CHARS = %r{(?:[[:word:]]|[-.?!:;À-ž/]|(?<=\w)\s(?=\w))}.freeze
```

Comment:
Should this perhaps match all Unicode letters in the context rather than just `a-z`?
https://rubular.com/r/dOnUgvJhli6Il2

Reply:
Yes, updated the regex it to match all Unicode letters and not just a-z.